### PR TITLE
updated filesystem to work with two servers

### PR DESCRIPTION
**Pull request by K-LV**

### CODE REVIEW

File: src/filesystem/File.js [hunk 1]
```diff
@@ -28,7 +28,11 @@
 define(function (require, exports, module) {
     "use strict";
     
-    var FileSystemEntry = require("filesystem/FileSystemEntry");
+    var FileSystemEntry = require("filesystem/FileSystemEntry"),
+    Content         = require("extensions/default/brackets-browser-livedev/nohost/src/content"),
+    Filer           = require("filesystem/impls/filer/BracketsFiler"),
```
Comment (humphd) [inline src/filesystem/File.js]: You're not actually using `Filer` here, just `Path`, so you can do `var Path = require("filesystem/impls/filer/BracketsFiler").Path;` and not do this and line 35.

File: src/filesystem/File.js [hunk 2]
```diff
@@ -145,6 +149,10 @@ define(function (require, exports, module) {
             
             callback = callback || function () {};
         }
+
+        if(Content.isHTML(Path.extname(this._path)) && data === "") {
```
Comment (humphd) [inline src/filesystem/File.js]: This line needs a comment above to explain why we're doing this.

**Comment by humphd** [comment]
A few things to address, r=me with those.  Get @gideonthomas to do a final pass over this when those are done.

**Comment by gideonthomas** [comment]
@K-LV this fixes the the issue I filed. But can you see if you can fix another subset of this issue:
When you add your link to the CSS file it works fine, but when you switch to the CSS file, make a change, save, and then return to your html file that has a link to the CSS file, it does not update the preview with the changes to the CSS file; you have to switch to another html file and then back to see the updated CSS.

**Comment by humphd** [comment]
That probably means that we need code to somehow update the URL we use for the CSS file in the link. With a normal URL, eg http://foo.com/style.css, if you update the content stored at the URL it won't affect the URL. But in our case we need to use a whole new URL now. The simplest way is to force a reload on the HTML page when a linked file changes, which will force the new bloburl to get used vs the old one.

On Mar 31, 2015 3:25 PM, Gideon Thomas <notifications@github.com> wrote:

@K-LV<https://github.com/K-LV> this fixes the the issue I filed. But can you see if you can fix another subset of this issue:
When you add your link to the CSS file it works fine, but when you switch to the CSS file, make a change, save, and then return to your html file that has a link to the CSS file, it does not update the preview with the changes to the CSS file; you have to switch to another html file and then back to see the updated CSS.

**Comment by gideonthomas** [comment]
agree! I think w should file a separate issue on it so that we can land this patch

**Pull request merged**